packages/wallet/.env

```diff
@@ -4,16 +4,18 @@ DEFAULT_GAS_PRICE=20000000000
 ENABLE_SOLC_OPTIMIZER=TRUE
 FIREBASE_PROJECT =  'rock-paper-scissors-dev'
 FIREBASE_API_KEY =  'AIzaSyAlGe17xjJjfoJ_KDYjCREg7ZL4ns61Chc'
+# TODO: Currently using existing channels fails so this is disabled
+# We need to address this
 # Controls if channels and related info are stored/loaded from IndexedDB
-USE_STORAGE=TRUE
+USE_STORAGE=FALSE
 
 # These values are required if contracts will be deployed to a test network
 # and should be overriden in a .env.local file
 ETH_ACCOUNT_MNENOMIC='A valid account mnemonic is required to deploy to a test network'
 INFURA_API_KEY='A valid Infura API key is required to deploy to a test network'
 
 GANACHE_HOST=0.0.0.0
-GANACHE_PORT=8560
+GANACHE_PORT=8547
```

Comment:
It's important that this stays different to the `SHARED_GANCHE_PORT` variable below. Otherwise other packages will think this is a shared ganache instance, but the wallet will be deploying to it without caching.